lib/comm_common.cpp

```diff
@@ -7,10 +7,11 @@
 
 #ifdef QUDA_BACKWARDSCPP
 #include "backward.hpp"
-namespace backward {
+namespace backward
+{
   static backward::SignalHandling sh;
 } // namespace backward
-#endif 
+#endif
 
 struct Topology_s {
```

Comment:
I might be missing the overall structure, but isn't this already defined in `communicator_quda.h`?